src/components/program/ProgramPage.jsx

```diff
@@ -39,15 +39,22 @@ class ProgramPage extends Component {
     const { programUUID } = this.props.pageContext;
     this.props.fetchUserProgramEnrollments();
     this.props.fetchProgramDiscussions(programUUID);
+    this.props.fetchProgramLiveSettings(programUUID);
   }
 
   componentDidUpdate(prevProps) {
-    const { enrolledPrograms, tabViewEnabled, programDiscussions } = this.props;
+    const {
+      enrolledPrograms,
+      tabViewEnabled,
+      programDiscussions,
+      liveSettings,
+    } = this.props;
     if (enrolledPrograms && enrolledPrograms !== prevProps.enrolledPrograms) {
       this.validateProgramAccess(enrolledPrograms);
     }
     if (tabViewEnabled && tabViewEnabled !== prevProps.tabViewEnabled) {
-      const switchTabView = programDiscussions.configured && tabViewEnabled;
+      // eslint-disable-next-line max-len
+      const switchTabView = tabViewEnabled && (programDiscussions.configured || liveSettings.configured);
```

Comment:
const switchTabView = tabViewEnabled 
&& (programDiscussions.configured || liveSettings.configured);

this will resolve the lint error. we do not need to disable the max len lint error.